_includes/header.html

```diff
@@ -33,8 +33,9 @@
       </div>
     </nav>
   </div>
-  <div class="donate-join-desktop">
+  <div class="donate-join-desktop flex-row align-items-center">
```

Comment:
Fixed alignment of Join Us button

After fix
![banner-after](https://user-images.githubusercontent.com/45829076/102859465-98f34500-4434-11eb-9868-b35801f46c32.jpg)

Before  fix
![banner-before](https://user-images.githubusercontent.com/45829076/102859426-90027380-4434-11eb-8605-b5385a2a0b7b.jpg)